app/helpers/notifications_helper.rb

```diff
@@ -25,6 +25,11 @@ module NotificationsHelper
     'RepositoryVulnerabilityAlert' => 'alert'
   }.freeze
 
+  SUBJECT_STATUS = {
+    success: "success",
+    failure: "failure"
+  }
```

Comment:
Can we also support `pending` and `error`?

Reply:
I did not add them because I didnt knew what to do with that state. What are we going to show to the user in these cases ??

Reply:
In the GitHub UI `pending` is shown as an orange dot and an `error` is shown as a red warning sign:

![github-checkout](https://user-images.githubusercontent.com/1060/46345336-17891f80-c63c-11e8-8b78-1637e18812be.png)

Reply:
So how will be differentiate between no status check types PR and PR where status checks are there and pending ??

Reply:
Wow, that's pretty shitty on GitHub's part but I think we can still make it work. 

Here's a status response for a commit with no statuses (still shows pending): https://gist.github.com/andrew/77c0d373361f2a650ec3df427eb98bca

and here's a status response for an actual pending: https://gist.github.com/andrew/a5c36620b2a83edf8c72d0086a3598ac

We need to check to see if there are any items in the `statuses` array, if so and the combined `state` is pending then we can set it to pending, otherwise (when `state` == pending and `statuses` == []) it should be set to `nil`.

Reply:
I still think that adding a pending status will not be too beneficial for Octobox users. In Octobox, PR as synced after a definite interval or forced sync, so most of the PR's will have a definite status i.e `Success/Failure` or rarely `error`.

Reply:
@andrew got it. thanks for the explanation.

Reply:
will also add filters and search for PR status.